### DISCUSSION TITLE
How can I contribute to this repository?

### DISCUSSION THREAD

**Discussion by dakshsethi** [Ayan-thecodeking/Here-Contributions-Begins#60] [Q&A] [answered]
Hi,
I want to contribute to this repository but I don't know how to do this. Can anyone please help me out?

**Comment by shital-punde** [comment, marked as answer]
First and foremost, you must choose a programming language of your choice. Once you are done with selecting a programming language, search for a project that is interesting to you. Try to work on as many issues as you can either across projects or for a single project.
Once you are comfortable with contributing to open-source, start participating in open source programs.
